README.md

```diff
@@ -34,14 +34,7 @@ pod 'LicensePlist'
 ### Homebrew (Also Recommended)
 
 ```sh
-brew install mono0926/license-plist/license-plist
-```
-
-Or
-
-```sh
-brew tap mono0926/license-plist
-brew install license-plist
+brew install licenseplist
```

Comment:
I'd like to keep to `license-plist`, not `licenseplist` 

Why did you change this?

Reply:
I followed the Homebrew's convention and your original naming was not followed it.

If the name of the repository is `mono0926/LicensePlist`, the name of the formula should be `licenseplist`, and if that of the repository is `mono0926/license-plist`, the name should be `license-plist`.

I don't know if Homebrew maintainers would accept this exception.

Reply:
@umireon 

I see, then I thought it would be okay 👌